arm-batch/2017-01-01/swagger/BatchManagement.json

```diff
@@ -1150,10 +1151,15 @@
         "id": {
```

Comment:
@ravbhatnagar Right now it is kind of late to apply the change since our service had already deployed. We will consider your recommendation with later breaking change.